### DISCUSSION TITLE
Releases stopped in January?

### DISCUSSION THREAD

**Discussion by mindplay-dk** [Alex313031/Thorium-Win-AVX2#134] [Q&A]
I discovered [this](https://twitter.com/lcasdev/status/1810696257137959018) today, and I'm looking to abandon Chrome.

This project looks interesting, but no releases since January?

Is this project still alive?

**Comment by gz83** [comment]
This project has not been abandoned

In addition, you should pay attention to the dynamics of this repo
https://github.com/Alex313031/Thorium

**Comment by mindplay-dk** [comment]
oh, so it's just this repository that's not in use anymore? the AVX2 build is coming from the main repo now? 🙂

**Reply by gz83** [reply]
Yeah

The main development happens in the main repo, but if you want to download the AVX2 version for windows, please go to the Thorium-win repo

**Comment by Alex313031** [comment]
@mindplay-dk Yes it's still alive. Currently working on updating it to M126,
